Parsers/ASimWebSession/Tests/Fortinet_FortiGate_ASimWebSession_DataTest.csv

```diff
@@ -0,0 +1,23 @@
+Result
+"(0) Error: 30 invalid value(s) (up to 10 listed) in 29949 records (99.83%) for field [DstHostname] of type [Hostname]: [""mask.icloud.com"",""mask-h2.icloud.com"",""mask-api.icloud.com""] (Schema:WebSession)"
```

Comment:
this error will get fixed if we use ASIM_ResolveDstFQDN function with DestinationHostName. This would set 4 fields Dst fields with right format. https://github.com/Azure/Azure-Sentinel/blob/master/ASIM/lib/functions/ASIM_ResolveDstFQDN.yaml

Reply:
You can use invoke with field name to call this function. e.g.
![image](https://github.com/user-attachments/assets/3c1734db-57ae-45d3-977e-93940d58ce68)

Reply:
please update in both ASim and vim files

Reply:
This has been added. Indeed it fixes the issue.